metricbeat/tests/system/test_haproxy.py

```diff
@@ -32,7 +32,7 @@ def test_info_socket(self):
         self.render_config_template(modules=[{
             "name": "haproxy",
             "metricsets": ["info"],
-            "hosts": ["tcp://%s:%d" % (os.getenv('HAPROXY_HOST', 'localhost'), 14567)],
+            "hosts": ["tcp://%s:%d" % (self.compose_hosts()[0], 14567)],
```

Comment:
What happens if more then 1 service is running? I think we are fine in this case as there will be only 1 haproxy instance but there are tests which require more then 1 service to be running.

Reply:
This method could return a hash with the service name as key, I'll refactor this. It could also return the hash with a list as values, in case some service has multiple replicas, e.g:
```
{
  "proxy": ["172.0.18.1"],
  "server": ["172.0.18.2", "172.0.18.3"],
}
```

I don't know if we have some case like this.

Reply:
Although if we have different service names for different versions this is not going to work as I'd want, I have to think more about this...